gcc/rust/resolve/rust-ast-resolve-unused.h

```diff
@@ -48,7 +58,7 @@ class ScanUnused
   {
     auto resolver = Resolver::get ();
     resolver->iterate_name_ribs ([&] (Rib *r) -> void { ScanRib (r); });
-    resolver->iterate_type_ribs ([&] (Rib *r) -> void { ScanRib (r); });
+    // resolver->iterate_type_ribs ([&] (Rib *r) -> void { ScanRib (r); });
```

Comment:
nice solution.

Reply:
Can you put a comment here saying why this was commented out, something like.

"ScanUnused is conflicting with the dead code analysis here on types"